migrations/20220919083117-add-password-to-shared-links.js

```diff
@@ -2,20 +2,20 @@
 
 module.exports = {
   async up(queryInterface, Sequelize) {
-    await queryInterface.addColumn('shares', 'password', {
+    await queryInterface.addColumn('shares', 'hashed_password', {
       type: Sequelize.BLOB('medium'),
```

Comment:
I think we can use 'text' and save it in base64. Any reason to use Blob?

Reply:
any, used blob to match with the password field on user, bus since  is the hashed there is no need indeed.